cmd/neofs-cli/modules/container/create.go

```diff
@@ -130,6 +130,7 @@ It will be stored in sidechain when inner ring will accepts it.`,
 				case <-ctx.Done():
 					common.ExitOnErr(cmd, "", errCreateTimeout)
 				case <-t.C:
+					common.ExitOnErr(cmd, "", common.ErrAwaitTimeout)
```

Comment:
this should be in `ctx.Done` case, and we should add context here instead of `ExitOnErr`
```suggestion
					common.ExitOnErr(cmd, "await container to be created", common.ErrAwaitTimeout)
```
same for `delete` and `set-eacl`